### PR TITLE
Feature/restyling

### PR DESCRIPTION
**Pull request by anhthu253**

### CODE REVIEW

File: components/ArticleCard.js [hunk 1]
```diff
@@ -36,30 +38,35 @@ export default function ArticleCard({ article, delible, onDelete }) {
...
-          <img
-            src={urlToImage}
-            alt="article image"
-            width="250"
-            height="150"
-          ></img>
 
-          <article>
+          <StyledArticle>
+            <img
```
Comment (Mark-Krensel) [inline components/ArticleCard.js:48]: you can use the ```Image``` Tag, if you want a responsive design

File: components/DragContainer.js [hunk 2]
```diff
@@ -31,17 +31,27 @@ export default function DragContainer({
...
   width: 100%;
-  height: 12rem;
-
+  --height: 11rem;
+  --background: #ccc;
+  display: flex;
+  list-style: none;
+  overflow-x: auto;
+  overflow-y: hidden;
   &:empty:not(:focus):before {
```
Comment (Mark-Krensel) [inline components/DragContainer.js:45]: don't know what all of that is, but it looks impressive&confusing at the same time

File: components/Layout.js [hunk 3]
```diff
@@ -1,33 +1,38 @@
...
-    <StyledLayout>
-      <Header />
-      <NavigationLayout></NavigationLayout>
-      <MainLayout>{children}</MainLayout>
-    </StyledLayout>
+    <>
+      <WallShred />
+      <StyledLayout>
+        <Header />
+        <NavigationBar></NavigationBar>
```
Comment (Mark-Krensel) [inline components/Layout.js:12]: If ```NavigationBar```doesn't have children, you can write it with a self closing tag

**Review by Mark-Krensel — APPROVED** [review]
seems like a lot of styling work, looking good - at least in the code ;)

File: components/Layout.js (hunk 3 above)
Comment (anhthu253) [reply components/Layout.js:12]: yeah that sprung out alot automatically and i dont care to change it anymore to self closing tag 😆

File: components/DragContainer.js (hunk 2 above)
Comment (anhthu253) [reply components/DragContainer.js:45]: i just copy that from google. It s the drag drop thing, simulating placeholder effect in input or textarea

File: components/ArticleCard.js (hunk 1 above)
Comment (anhthu253) [reply components/ArticleCard.js:48]: Unfortunately I can not use Image here. The images are from everywhere, depending on the article so I dont have control over the domains :(

**Pull request merged**